database/scrapers/buildings.ts

```diff
@@ -96,13 +82,14 @@ export default class BuildingsScraper extends BaseScraperModule {
           latitude: data.latitude,
           longitude: data.longitude,
           haveFood: data.food ?? false,
-          coverKey: await this.semaphore.runTask(() =>
-            this.uploadCoverAndGetKey(data),
-          ),
+          coverKey:
+            data.cover === null
+              ? null
+              : await this.directusUploadFieldAndGetKey(data.cover),
```

Comment:
use `.addErrorContext` here to include something about the building here (either id or name), so we know which one failed.